tests/end_to_end_tests.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+This enables to parameterize end to end tests - the tests are run by Travis each time you commit to the github repo
+"""
+
+import pytest
+import pandas as pd
+import subprocess
+import sys
+sys.path.append("..")
```

Comment:
What is this for? Kind of mysterious adding two dots

Reply:
This script is in the /test folder, but we need to access other modules (like the `constants.py` module) that are located in the parent folder, thus we need to add the parent folder ".." to the path. This might not be the cleanest way to do it but it works !